client/payment-details/payment-method/afterpay-clearpay/index.tsx

```diff
@@ -30,22 +32,38 @@ const formatPaymentMethodDetails = ( charge ) => {
 };
 
 /**
- * Placeholders to display while loading.
+ * Placeholder object for loading state
  */
-const paymentMethodPlaceholders = {
+const paymentMethodPlaceholders: PaymentMethodDetails = {
 	id: 'id placeholder',
 	name: 'name placeholder',
 	email: 'email placeholder',
 	formattedAddress: 'address placeholder',
 };
 
-const CardDetails = ( { charge = {}, isLoading } ) => {
-	const details =
+/**
+ * Props interface for AfterpayClearpayDetailss component
+ */
+interface AfterpayClearpayDetailsProps {
+	charge?: Charge;
+	isLoading: boolean;
+}
```

Comment:
Again, this interface is replicated: it appears like we could create some shared `AffirmAfterpayDetailsProps` interface and import it in both files, unless there is some reason to suspect that these two implementations (Affirm & Afterpay/Clearpay) might require a separate, unique interface at some point in the future. Thoughts?

Reply:
On further more considered reflection, these two files are basically an entirely duplicated implementation--even the React components themselves appear identical. 😅 Perhaps solving this duplication could be a task for a separate refactor. Let me know if you'd rather handle this in a separate issue instead of resolving it here. 🙏